clusters/app.ci/assets/vrutkovs-pj-access.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: vrutkovs-prowjob-access
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: prowjob-create
```

Comment:
I guess you'd want one to `prowjob-read` too? Feel free to unhold if not.

/approve
/hold

Reply:
`cluster-bot` needs permissions to create new prowjobs

Reply:
Ah, correct. Added CRB for that too